src/esp/scene/SceneGraph.h

```diff
@@ -27,12 +27,11 @@ class SceneGraph {
   SceneNode& getRootNode() { return rootNode_; }
   const SceneNode& getRootNode() const { return rootNode_; }
 
-  // TODO: remove this
-  gfx::DrawableGroup& getDrawables() {
-    return drawableGroups_.at(std::string{});
+  gfx::DrawableGroup& getDrawables(const char* groupName = "") {
```

Comment:
Why char pointers as args instead of std::string?

Reply:
I am passing a string literal to this function, so I use `const char*` here. It would be faster if `std::string` can be avoided. 
But here it will be converted to the `std::string` anyway, so I guess you are right, I can just use string directly.

Reply:
@bigbike Couldn't you use corrade::stringview or something to avoid a copy?

Reply:
>corrade::stringview 

@Skylion007 : No, that is exactly for the C string literals. To avoid the copy then the data structure should not use `std::string` as the key which I doubt.